sway/tree/container.c

```diff
@@ -826,6 +827,7 @@ void container_set_floating(struct sway_container *container, bool enable) {
 	struct sway_seat *seat = input_manager_current_seat();
 	struct sway_workspace *workspace = container->pending.workspace;
 	struct sway_container *focus = seat_get_focused_container(seat);
+	struct sway_xdg_decoration *deco = container->view->xdg_decoration;
```

Comment:
Hm. I forgot one thing: at this point, there's no guarantee that the container has a view. Se we can't have this variable declaration here, it needs to be moved down the `if`s.